src/opentaskpy/addons/gcp/remotehandlers/creds.py

```diff
@@ -0,0 +1,35 @@
+"""GCP helper functions."""
+
+import opentaskpy.otflogging
+from google.auth.transport.requests import Request
+from google.oauth2 import credentials, service_account
+from opentaskpy.exceptions import RemoteTransferError
+
+
+def get_access_token(credentials: dict) -> credentials:
+    """Get an access token for GCP using the provided credentials.
+
+    Args:
+        credentials: The credentials to use
+    """
+    logger = opentaskpy.otflogging.init_logging(__name__, None, None)
+    try:
+        # Get an Access Token
+        authCreds = None  # Initialising authCreds
```

Comment:
https://peps.python.org/pep-0008/#descriptive-naming-styles - We're using `lower_case_with_underscores`, even though it's not technically a reserved keyword, you can add an `_` after `credentials` (which is what you named it before) to avoid the overriding variable error you had before